wpilibNewCommands/src/test/native/cpp/frc2/command/WaitCommandTest.cpp

```diff
@@ -27,3 +27,60 @@ TEST_F(WaitCommandTest, WaitCommandSchedule) {
 
   frc::sim::ResumeTiming();
 }
+
+TEST_F(WaitCommandTest, WaitCommandScheduleLambda) {
+  frc::sim::PauseTiming();
+
+  CommandScheduler scheduler = GetScheduler();
+
+  WaitCommand command([] { return 100_ms; });
+
+  scheduler.Schedule(&command);
+  scheduler.Run();
+  EXPECT_TRUE(scheduler.IsScheduled(&command));
+  frc::sim::StepTiming(110_ms);
+  scheduler.Run();
+  EXPECT_FALSE(scheduler.IsScheduled(&command));
+
+  frc::sim::ResumeTiming();
+}
+
+TEST_F(WaitCommandTest, WaitCommandScheduleNonStaticLambda) {
```

Comment:
What about this test is non-static? For the duration of each scheduling of the command, the lambda returns the same value. This intention doesn't match the implementation being tested.

Reply:
that should be fixed now